hubblestack/extmods/returners/slack_pulsar_returner.py

```diff
@@ -261,33 +261,33 @@ def returner(ret):
     api_key = _options.get('api_key')
 
     if not channel:
-        log.error('slack_pulsar.channel not defined in salt config')
-        return
+        LOG.error('slack_pulsar.channel not defined in salt config')
+        return None
 
     if not username:
-        log.error('slack_pulsar.username not defined in salt config')
-        return
+        LOG.error('slack_pulsar.username not defined in salt config')
+        return None
 
     if not as_user:
-        log.error('slack_pulsar.as_user not defined in salt config')
-        return
```

Comment:
`return None` is something the linter recommends?

Reply:
It doesn't like that some return statements return something and others don't, it says all of them should return something or none of them should. I figured returning None would be the fix for that, should I have it return something else instead?

Reply:
Oh, right. I've seen that before now that you say it like that.  No, it's definitely right the way it is, it just seemed odd to me.